RPLCD/lcd.py

```diff
@@ -450,6 +462,11 @@ def create_char(self, location, bitmap):
         # Restore cursor pos
         self.cursor_pos = pos
 
+    def backlight(self, value):
+        """Turn backlight GPIO on or off."""
+        if self.pins.backlight != None:
+            GPIO.output(self.pins.backlight, bool(value) ^ bool(self.backlight_active_low))
+
```

Comment:
This would probably make more sense as a property, right after `display_enabled` (around line 331):

```python
    def _get_backlight_enabled(self):
        # We could probably read the current GPIO output state via sysfs, but
        # for now let's just store the state in the class
        if self.pins.backlight is None:
            raise ValueError('You did not configure a GPIO pin for backlight control!')
        return bool(self._backlight_enabled)

    def _set_backlight_enabled(self, value):
        if self.pins.backlight is None:
            raise ValueError('You did not configure a GPIO pin for backlight control!')
        if not isinistance(value, bool):
            raise ValueError('backlight_enabled must be set to ``True`` or ``False``.')
        self._backlight_enabled = value
        GPIO.output(self.pins.backlight, value ^ (self.backlight_mode is BacklightMode.active_low))

    backlight_enabled = property(_get_backlight_enabled, _set_backlight_enabled,
            doc='Whether or not to turn on the backlight.')
```

It's a more consistent API that allows you to simply assign `True` and `False` to `lcd.backlight_enabled`.